README.md

```diff
@@ -15,67 +17,21 @@ For each of the three exchanges `USD`, `EUR` and `JPY`, this demo makes use of t
 - `CrytoTraderQuotes{USD/EUR/JPY}`
 - `CryptoTraderQuotesAvg{USD/EUR/JPY}`
 
-The below steps will describe on how to deploy the node and UI application.
+The below steps will describe on how set up the frontend.
 
-# 2. Prerequisites
+# 1. Prerequisites
 
-Clone tutorial-cryptotrading.
+Clone or Fork this repo.
 
 `nodejs` and `npm` must be installed on your system.
 
-`trades` collection should be present.
-
-# 3. How to run app(UI) locally
+# 2. How to run app(UI) locally
 
-> NOTE: This step is just for running the UI locally. The actual app is deployed on an AWS S3 Bucket. For the steps on S3 go to the `How to deploy app on S3` section.
+> NOTE: This step is just for running the UI locally.
 
 Go to crypto-demo and run the following
```

Comment:
How am I running it? I don't see anything to copy and paste. If the line below is what I'm using, then it should be in a code block.